src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -895,6 +895,9 @@
     <value>Font face</value>
     <comment>Name for a control to select the font for text in the app.</comment>
   </data>
+  <data name="Profile_FontFace.HelpText" xml:space="preserve">
+    <value>You can use multiple fonts by separating them with an ASCII comma.</value>

```

Comment:
hmmm can we make this smarter? we can just say "A comma" and then in the code transform anything that looks like a comma into an ASCII comma?

Reply:
That would probably make the life in certain languages easier, but I don't believe such a database is being maintained anywhere. We could then only do it for certain languages. Additionally, a Japanese comma, etc., is technically a valid part of a font name.

This reminds me that I should escape font names with quotes or commas in them. Huh.